src/Nancy.Demo.Authentication.Forms.TestingDemo/LoginFixture.cs

```diff
@@ -17,10 +17,10 @@ public LoginFixture()
         }
 
         [Fact]
-        public void Should_redirect_to_login_with_error_querystring_if_username_or_password_incorrect()
+        public async Task Should_redirect_to_login_with_error_querystring_if_username_or_password_incorrect()
         {
             // Given, When
-            var response = browser.Post("/login/", (with) =>
+            var response = await browser.Post("/login/", (with) =>

```

Comment:
Not 100% sure this is wise - it's a sneaky breaking change because stuff may well still build, it just (potentially) won't work properly.

We are planning on bringing in the LegacyNancyModule and making NancyModule async only, perhaps we could do a similar thing here?